### PR TITLE
add go reoutines

### PR DESCRIPTION
**Pull request by moogacs**
Happy hacking :)

### CODE REVIEW

**Comment by sagar-jadhav** [comment]
Thank you @moogacs for contribution 👍

**Comment by sagar-jadhav** [comment]
@aprabhat & @rajkumarGosavi Can you please review the code changes?

File: src/routine.go [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+		fmt.Println(from, ":", i)
+	}
+}
+
+func main() {
+	printFrom("direct")
+	go printFrom("goroutine")
+	go func(msg string) {
+		fmt.Println(msg)
+	}("going")
```
Comment (rajkumarGosavi) [inline src/routine.go:19]: Please write beautiful comments about why we need to pass the context to go routine. It is an important topic
also an example where a go routine is called in a for loop will be very helpful something like
```

for ...{

    go someFunc(){

}(context)

}

```

also show how one can use multiple procs using **GOMAXPROCS**

**Review by rajkumarGosavi — CHANGES_REQUESTED** [review]

**Comment by sagar-jadhav** [comment]
@moogacs Can you please have look on review comments & incorporate suggested changes?

**Comment by moogacs** [comment]
@sagar-jadhav  will do that

**Comment by sagar-jadhav** [comment]
Hello @moogacs any updates on  **incorporating review comments**? Let's try to close this as early as possible. 
### 
Let me know if you require any help from our side.

**Comment by sagar-jadhav** [comment]
@moogacs I am closing #78 PR due to inactivity. Please raise another PR once you ready with your changes.

**Pull request closed without merging**